### ISSUE TITLE
How do I NOT allow users to type in dates into the two date fields - only select from the calendar ?

### ISSUE THREAD

**Issue by bodhidg** [airbnb/react-dates#2149]
I want to disallow users from typing out the start & end dates in the date range picker - and only select dates from the calendar. How do i achieve that ?

**Comment by ljharb** [comment]
I _hope_ there's no easy way to achieve that - that would violate accessibility guidelines and discriminate against users with visual difficulties, which in some countries (like the US) could earn you millions of dollars in civil penalties in court.

You may be able to figure out a way to make it work, but I strongly advise against it.

**Comment by bodhidg** [comment]
That sounds scary. Thanks @ljharb - will avoid doing this.

**Comment by ljharb** [comment]
I'm glad I came across sufficiently scary, that was my goal :-) i hope i didn't imply you had any malicious intent, though!

**Comment by bodhidg** [comment]
> I'm glad I came across sufficiently scary, that was my goal :-) i hope i didn't imply you had any malicious intent, though!

Absoutely. Thanks for the details. I have recently started picking up React and I am realizing I have barely scratched the surface - so a lot of my questions sound really stupid : )
